File: models/company_model.go
package models

// Company struct
type Company struct {
	ID       int64  `orm:"auto;pk;type(bigint);column(id)" json:"id"`      // id
	Title    string `orm:"null;type(char);column(title)" json:"title"`     // 公司名称
	Logo     string `orm:"null;type(char);column(logo)" json:"logo"`       // 公司logo
	Service  string `orm:"null;column(service)" json:"service"`            // 在线客服服务时间
	Email    string `orm:"null;column(email)" json:"email"`                // 公司邮箱
	Tel      string `orm:"null;column(tel)" json:"tel"`                    // 公司电话
	Address  string `orm:"null;type(char);column(address)" json:"address"` // 公司地址
	UpdateAt int64  `orm:"type(bigint);column(update_at)" json:"update_at"`
}

File: ui/kefu_workbench/lib/models/company_model.dart
class CompanyModel {
  int id;
  String title;
  String logo;
  String service;
  String email;
  String tel;
  String address;
  int updateAt;

  CompanyModel(
      {this.id,
      this.title,
      this.logo,
      this.service,
      this.email,
      this.tel,
      this.address,
      this.updateAt});

  CompanyModel.fromJson(Map<String, dynamic> json) {
    id = json['id'];
    title = json['title'];
    logo = json['logo'];
    service = json['service'];
    email = json['email'];
    tel = json['tel'];
    address = json['address'];
    updateAt = json['update_at'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['id'] = this.id;
    data['title'] = this.title;
    data['logo'] = this.logo;
    data['service'] = this.service;
    data['email'] = this.email;
    data['tel'] = this.tel;
    data['address'] = this.address;
    data['update_at'] = this.updateAt;
    return data;
  }
}